sourced/ml/models/model_converters/merge_bow.py

```diff
@@ -0,0 +1,69 @@
+import os
+from scipy.sparse import hstack, vstack
+
+from sourced.ml import extractors
+from sourced.ml.models.bow import BOW
+from sourced.ml.models.model_converters.base import Model2Base
+
+
+class MergeBOW(Model2Base):
+    """
+    Merges several :class:`DocumentFrequencies` models together.
+    """
+    MODEL_FROM_CLASS = BOW
+    MODEL_TO_CLASS = BOW
+
+    def __init__(self, features=None, *args, **kwargs):
+        super().__init__(num_processes=1, *args, **kwargs)
+        self.documents = None
+        self.tokens = None
+        self.matrix = None
+        self.deps = None
+        self.features = None
+        if features:
+            self.features = [ex.NAMESPACE for ex in extractors.__extractors__.values()
```

Comment:
better to rename to `features_namespaces`
otherwise, it confuses names and namespaces

Reply:
agreed